.drone/drone.yml

```diff
@@ -0,0 +1,83 @@
+---
+image_pull_secrets:
+- dockerconfigjson
+kind: pipeline
+name: pr
+steps:
+- commands:
+  - go test ./...
+  image: golang:1.17
+  name: test
+  settings: {}
+- commands:
+  - echo -n "${DRONE_SOURCE_BRANCH}-${DRONE_COMMIT_SHA}" | tr "/" "_" > .tags
+  image: golang:1.17
+  name: generate tags
+  settings: {}
+- commands: []
+  image: plugins/gcr
+  name: build
+  settings:
+    json_key:
+      from_secret: gcr_admin
+    registry: us.gcr.io
+    repo: kubernetes-dev/influx2cortex
+trigger:
+  event:
+    include:
+    - pull_request
+type: docker
+---
+image_pull_secrets:
+- dockerconfigjson
+kind: pipeline
+name: master
+steps:
+- commands:
+  - go test ./...
+  image: golang:1.17
+  name: test
+  settings: {}
+- commands:
+  - echo -n "${DRONE_BRANCH}-${DRONE_COMMIT_SHA},latest" > .tags
+  image: golang:1.17
+  name: generate tags
+  settings: {}
+- commands: []
+  image: plugins/gcr
+  name: build + push
+  settings:
+    json_key:
+      from_secret: gcr_admin
+    registry: us.gcr.io
+    repo: kubernetes-dev/influx2cortex
+trigger:
+  branch:
+  - master
+  event:
+    include:
+    - push
```

Comment:
so am I right that this pushes to docker, but does not cause that newly pushed image to be deployed?

Reply:
Yep, specifically it'll push the image [here](https://console.cloud.google.com/gcr/images/kubernetes-dev/us/influx2cortex?project=kubernetes-dev).